tools/semgrep-py.yml

```diff
@@ -81,6 +81,54 @@ rules:
         - zerver/migrations/0493_rename_userhotspot_to_onboardingstep.py
         - pgroonga/migrations/0002_html_escape_subject.py
 
+  - id: use-addindexconcurrently
+    pattern-either:
+      - pattern: AddIndex(...)
+      - pattern: migrations.AddIndex(...)
```

Comment:
Semgrep understands imports. It’s more accurate and simpler to specify `django.db.migrations.AddIndex(...)` as the only pattern.